becquerel/parsers/iec1455.py

```diff
@@ -95,7 +95,7 @@ def read(filename, verbose=False, cal_kwargs=None):
     energy_channel_pairs = []
     energy_res_pairs = []  # currently unused
     energy_eff_pairs = []  # currently unused
-    with Path(filename).open() as f:
+    with Path(filename).open(encoding="utf-8") as f:
```

Comment:
is this really necessary?

Reply:
This is [`PLW1514`](https://docs.astral.sh/ruff/rules/unspecified-encoding/).